Tasks/AzureSpringCloudV0/task.loc.json

```diff
@@ -18,7 +18,7 @@
   "preview": true,
   "version": {
     "Major": 0,
-    "Minor": 201,
+    "Minor": 204,
```

Comment:
Could you please bump task version to 205 as current sprint number?

Reply:
Ok, I have bumped the task version.